duckduckgo.pot

```diff
@@ -3391,6 +3391,11 @@ msgctxt "mobile expanded map"
 msgid "Expand Map"
 msgstr ""
 
+# smartling.placeholder_format=C
+#. A prefix text that can be used before numbers to indicate the time remaining before something expires. E.g 'Expires in 3 days'.
+msgid "Expires in"
+msgstr ""
+
```

Comment:
Please, include all elements of the sentence in one token and use placeholders. Some languages need to reorder the elements of the sentence, and if the elements are broken up in multiple parts translators can’t do that. 

On the SERP, unfortunately, we can’t handle plurals properly, so for now you need to create two strings, one for the singular, one for the plurals (languages with multiple plurals unfortunately have to make do).

You can find tips for better translations tokens here: https://app.asana.com/0/904401899170/1200205345312078. 

I suggest you close this PR, make the changes in dub and re-run `just upload-translations`.

cc @franfaccin This is a relatively common mistake we should catch at review time in dub 🙂.

Reply:
@dbajpeyi I can't find the translation tokens in the original PR. I just want to double check if you've pushed the changes to text.json already or just pushed the translations from local changes?
@GioSensation - Maybe we need to update the docs with suggestions on how to run translations if we want them separate from the original PR?

Reply:
@GioSensation That makes sense with the ordering in different languages 🤦 - I will re-do this PR with the changes in dub, and create another one with two forms of the sentences.

@franfaccin They were not committed yet, they're from my local changes.

Reply:
@dbajpeyi Cool, no worries. For the future, it's better to have the strings reviewed in dub before uploading, because uploading here triggers the translation job immediately.

I am going to close the job in Smartling, too, since some of the strings will be slightly different.